### PR TITLE
Documentation release v3.7.0

### PR DESCRIPTION
**Pull request by ann0see**
Still missing:

1. French and Italian translation

Almost finished:

Spanish translation

### CODE REVIEW

**Comment by ann0see** [comment]
Doesn't seem to be bad this time.

http://127.0.0.1:4000/de/kb/2020/03/28/Server-Rpi.html
- (241:7) 'Get the ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (241:102) 'dependen..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)
- (244:164) 'default ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)
- (299:12) 'KB Übers..' => http://127.0.0.1:4000/de/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/de/kb (301)
    - http://127.0.0.1:4000/de/kb/ (200)

http://127.0.0.1:4000/de/kb/2020/09/20/Linux-Install-Script.html
- (382:12) 'KB Übers..' => http://127.0.0.1:4000/de/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/de/kb (301)
    - http://127.0.0.1:4000/de/kb/ (200)

http://127.0.0.1:4000/de/kb/2020/10/19/Software-Synth.html
- (333:12) 'KB Übers..' => http://127.0.0.1:4000/de/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/de/kb (301)
    - http://127.0.0.1:4000/de/kb/ (200)

http://127.0.0.1:4000/de/kb/2020/11/24/Multiple-Audio-Interfaces.html
- (286:12) 'KB Übers..' => http://127.0.0.1:4000/de/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/de/kb (301)
    - http://127.0.0.1:4000/de/kb/ (200)

http://127.0.0.1:4000/de/kb/2021/01/05/Jamulus-Sound-Devices.html
- (550:12) 'KB Übers..' => http://127.0.0.1:4000/de/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/de/kb (301)
    - http://127.0.0.1:4000/de/kb/ (200)

http://127.0.0.1:4000/de/wiki/Client-Troubleshooting
- (194:101) 'ASIO4LL ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Command-Line-Options
- (353:29) 'Tipps & ..' => http://127.0.0.1:4000/de/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Hardware-Setup
- (248:22) 'client i..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Server-Linux
- (247:6) 'Get the ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (247:92) 'dependen..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)
- (429:248) 'More inf..' => http://127.0.0.1:4000/de/wiki/Compiling#the-headless-build-flag (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Server-Win-Mac
- (254:99) 'private ..' => http://127.0.0.1:4000/de/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)

http://127.0.0.1:4000/es/kb/2020/03/28/Server-Rpi.html
- (241:7) 'Get the ..' => http://127.0.0.1:4000/es/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (241:102) 'dependen..' => http://127.0.0.1:4000/es/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)
- (241:311) 'headless..' => http://127.0.0.1:4000/es/wiki/Server-Linux#running-a-headless-server (HTTP 200 but missing anchor)
- (244:51) 'these in..' => http://127.0.0.1:4000/es/wiki/Server-Linux#compile-sources-create-a-user (HTTP 200 but missing anchor)
- (244:164) 'default ..' => http://127.0.0.1:4000/es/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)
- (256:151) 'Linux gu..' => http://127.0.0.1:4000/es/wiki/Server-Linux#create-a-start-script (HTTP 200 but missing anchor)
- (299:12) 'Resumen BC' => http://127.0.0.1:4000/es/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/es/kb (301)
    - http://127.0.0.1:4000/es/kb/ (200)

http://127.0.0.1:4000/es/kb/2020/09/20/Linux-Install-Script.html
- (382:12) 'Resumen BC' => http://127.0.0.1:4000/es/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/es/kb (301)
    - http://127.0.0.1:4000/es/kb/ (200)

http://127.0.0.1:4000/es/kb/2020/10/19/Software-Synth.html
- (333:12) 'Resumen BC' => http://127.0.0.1:4000/es/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/es/kb (301)
    - http://127.0.0.1:4000/es/kb/ (200)

http://127.0.0.1:4000/es/kb/2020/11/24/Multiple-Audio-Interfaces.html
- (286:12) 'Resumen BC' => http://127.0.0.1:4000/es/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/es/kb (301)
    - http://127.0.0.1:4000/es/kb/ (200)

http://127.0.0.1:4000/es/kb/2021/01/05/Jamulus-Sound-Devices.html
- (550:12) 'Resumen BC' => http://127.0.0.1:4000/es/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/es/kb (301)
    - http://127.0.0.1:4000/es/kb/ (200)

http://127.0.0.1:4000/es/wiki/Server-Linux
- (247:470) 'ver la n..' => http://127.0.0.1:4000/es/wiki/Server-Linux#qu%C3%A9-hace-el-indicador-de-compilacion-headless (HTTP 200 but missing anchor)

http://127.0.0.1:4000/fr/kb/2020/03/28/Server-Rpi.html
- (237:7) 'Get the ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (237:102) 'dependen..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)
- (237:311) 'headless..' => http://127.0.0.1:4000/fr/wiki/Server-Linux#running-a-headless-server (HTTP 200 but missing anchor)
- (240:51) 'these in..' => http://127.0.0.1:4000/fr/wiki/Server-Linux#compile-sources-create-a-user (HTTP 200 but missing anchor)
- (240:164) 'default ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)
- (252:151) 'Linux gu..' => http://127.0.0.1:4000/fr/wiki/Server-Linux#create-a-start-script (HTTP 200 but missing anchor)
- (295:12) 'KB overv..' => http://127.0.0.1:4000/fr/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/fr/kb (301)
    - http://127.0.0.1:4000/fr/kb/ (200)

http://127.0.0.1:4000/fr/kb/2020/09/20/Linux-Install-Script.html
- (378:12) 'KB overv..' => http://127.0.0.1:4000/fr/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/fr/kb (301)
    - http://127.0.0.1:4000/fr/kb/ (200)

http://127.0.0.1:4000/fr/kb/2020/10/19/Software-Synth.html
- (329:12) 'KB overv..' => http://127.0.0.1:4000/fr/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/fr/kb (301)
    - http://127.0.0.1:4000/fr/kb/ (200)

http://127.0.0.1:4000/fr/kb/2020/11/24/Multiple-Audio-Interfaces.html
- (282:12) 'KB overv..' => http://127.0.0.1:4000/fr/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/fr/kb (301)
    - http://127.0.0.1:4000/fr/kb/ (200)

http://127.0.0.1:4000/fr/kb/2021/01/05/Jamulus-Sound-Devices.html
- (546:12) 'KB overv..' => http://127.0.0.1:4000/fr/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/fr/kb (301)
    - http://127.0.0.1:4000/fr/kb/ (200)

http://127.0.0.1:4000/fr/wiki/Command-Line-Options
- (301:56) 'paramétr..' => http://127.0.0.1:4000/fr/wiki/Server-Win-Mac#%23enregistrement (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/kb/2020/03/28/Server-Rpi.html
- (258:7) 'Get the ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (258:102) 'dependen..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)
- (261:164) 'default ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)
- (316:12) 'KB overv..' => http://127.0.0.1:4000/it/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/it/kb (301)
    - http://127.0.0.1:4000/it/kb/ (200)

http://127.0.0.1:4000/it/kb/2020/09/20/Linux-Install-Script.html
- (399:12) 'KB overv..' => http://127.0.0.1:4000/it/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/it/kb (301)
    - http://127.0.0.1:4000/it/kb/ (200)

http://127.0.0.1:4000/it/kb/2020/10/19/Software-Synth.html
- (350:12) 'KB overv..' => http://127.0.0.1:4000/it/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/it/kb (301)
    - http://127.0.0.1:4000/it/kb/ (200)

http://127.0.0.1:4000/it/kb/2020/11/24/Multiple-Audio-Interfaces.html
- (303:12) 'KB overv..' => http://127.0.0.1:4000/it/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/it/kb (301)
    - http://127.0.0.1:4000/it/kb/ (200)

http://127.0.0.1:4000/it/kb/2021/01/05/Jamulus-Sound-Devices.html
- (567:12) 'KB overv..' => http://127.0.0.1:4000/it/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/it/kb (301)
    - http://127.0.0.1:4000/it/kb/ (200)

http://127.0.0.1:4000/it/wiki/Client-Troubleshooting
- (211:101) 'ASIO4LL ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Command-Line-Options
- (256:32) 'See note' => http://127.0.0.1:4000/it/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)
- (370:28) 'Tips & T..' => http://127.0.0.1:4000/it/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Getting-Started
- (224:336) 'hardware..' => http://127.0.0.1:4000/it/wiki/Getting-Started#recommended-hardware (HTTP 200 but missing anchor)
- (273:186) 'il parag..' => http://127.0.0.1:4000/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Hardware-Setup
- (265:22) 'client i..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Installation-for-Linux
- (305:47) 'ottenere..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (305:135) 'istruzio..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Onboarding
- (221:74) 'ASIO4ALL..' => http://127.0.0.1:4000/it/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Running-a-Server
- (230:241) 'Rule Num..' => http://127.0.0.1:4000/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)
- (232:373) 'follow R..' => http://127.0.0.1:4000/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Server-Linux
- (264:6) 'Get the ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (264:92) 'dependen..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)

http://127.0.0.1:4000/it/wiki/Server-Win-Mac
- (271:99) 'private ..' => http://127.0.0.1:4000/it/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)

http://127.0.0.1:4000/kb/2020/03/28/Server-Rpi.html
- (299:12) 'KB overv..' => http://127.0.0.1:4000/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/kb (301)
    - http://127.0.0.1:4000/kb/ (200)

http://127.0.0.1:4000/kb/2020/09/20/Linux-Install-Script.html
- (382:12) 'KB overv..' => http://127.0.0.1:4000/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/kb (301)
    - http://127.0.0.1:4000/kb/ (200)

http://127.0.0.1:4000/kb/2020/10/19/Software-Synth.html
- (333:12) 'KB overv..' => http://127.0.0.1:4000/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/kb (301)
    - http://127.0.0.1:4000/kb/ (200)

http://127.0.0.1:4000/kb/2020/11/24/Multiple-Audio-Interfaces.html
- (286:12) 'KB overv..' => http://127.0.0.1:4000/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/kb (301)
    - http://127.0.0.1:4000/kb/ (200)

http://127.0.0.1:4000/kb/2021/01/05/Jamulus-Sound-Devices.html
- (550:12) 'KB overv..' => http://127.0.0.1:4000/kb (HTTP 200)
  - redirect path:
    - http://127.0.0.1:4000/kb (301)
    - http://127.0.0.1:4000/kb/ (200)

http://127.0.0.1:4000/wiki/Command-Line-Options
- (353:28) 'Tips & T..' => http://127.0.0.1:4000/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)


Stats:
   12844 links
     283 destination URLs
     615 URLs ignored
      42 warnings
       0 errors

**Comment by ann0see** [comment]
It would be great if every translator would look at this (huge) diff and check if everything looks ok here. We'll have a release soon.

**Comment by ann0see** [comment]
@jamulussoftware/trreviewers We're moving towards a final release of the Jamulus application soon. The release candidate is on it's way! It would be great if you check if there are any typos/wrong links here!

**Comment by trebmuh** [comment]
@ann0see : what is the ETA for the 3.7.0 release?
I've been way from the internet on the 2 last weeks and am only back there since yesterday, so I might have missed a few messages.

**Comment by ann0see** [comment]
Not exactly sure. The release candidate will be published today, but you can always push to release after this one is merged.

**Comment by ann0see** [comment]
There's no need to rush anything on your side ;-).

**Comment by ignotus666** [comment]
@ann0see: what exactly are we meant to look for in that list you posted above? I'm guessing it's detecting some kind of error in links but I don't understand what sort of errors they are. These automated checkers often seem to whine about non-issues so it's hard to tell when the complaint is valid.

**Comment by ann0see** [comment]
It's mainly about checking if any anchors aren't set. 

The only (not confirmed) broken anchor for Spanish should be this one:
image

**Comment by ignotus666** [comment]
Just checked - it looks ok to me, it follows the exact same format as the English version.

Edit: wait - it was missing a tilde. Could that be it?

**Comment by ann0see** [comment]
Yep. That could be a problem

**Comment by ann0see** [comment]
@ignotus666 @trebmuh 

The pages Linux-Install-Script; Multiple-Audio-Interfaces, Server-Rpi; Software-Synth and Sound-Devices have been moved to the knowledge base. I'd not focus too much on translating them, but you can of course. Should I delete them from the translate3_7_0 branch now?

**Comment by trebmuh** [comment]
> Should I delete them from the translate3_7_0 branch now?

If they are not to be translated anymore, please do.

**Comment by ann0see** [comment]
You can of course translate them if you want to ;-). I'll delete the french page soon.

**Comment by ann0see** [comment]
Ok. It's gone.

File: 1-index.html [hunk 1]
```diff
@@ -16,11 +16,7 @@
   <div class="fx-col-100-xs">
     <div itemprop="abstract">
       <h2>What is Jamulus?</h2>
-      Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. Use your Windows, macOS or Linux machine to connect to Jamulus servers worldwide, for free over a normal broadband
-      connection. Or
-      host your own private server. Jamulus is designed for high quality, low-latency sound, making it easy to play together remotely and in time. Have a look at <a href="wiki/Demos" target="_blank"
-        rel="noreferrer">how people
-        do it</a>!
+      Jamulus is software for playing music, rehearsing, or just jamming with anyone online with low latency. You can use your Windows, macOS or Linux machine to connect to Jamulus servers worldwide. Jamulus is free and you can just use your normal broadband connection. Simply connect to a public server or host your own private one. Jamulus has been in development since 2006 and is designed for high quality, low-latency sound, making it easy to play together remotely and in time. Have a look at <a href="wiki/Demos" target="_blank" rel="noreferrer">how people do it</a>!
```
Comment (gilgongo) [inline 1-index.html:19]: I'd remove the initial "with low latency" as being too geeky for a lead-in line. It's mentioned at the end, no problem.

File: 1-index.html [hunk 2]
```diff
@@ -34,7 +30,7 @@ <h2>What is Jamulus?</h2>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
      <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to sing, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
+    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
```
Comment (gilgongo) [inline 1-index.html:33]: This rather kills the slightly poetic nature of the original :-) What's wrong with choirs singing? And do rock bands no rehearse? That's not the point, I know, but...

File: wiki/en/en-Command-Line-Options.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ You can see all possible options your version supports by starting Jamulus with
 |    `-c` |`--connect`        | Connect to given server address on startup, format `address[:port]` | (client only) |
 |    `-d` |`--discononquit`   | Disconnect all clients on quit | (server only) |
 |    `-e` |`--centralserver`  | Makes the server public and sets its genre. See also `-o`| (server only) [See list](Central-Servers). |
-|    `-f` |`--listfilter`     | Whitelist servers registering on the serverlist, format `ip address 1[;ip address 2]` | (central server only) [See note](Choosing-a-Server-Type#3-central). |
+|    `-f` |`--listfilter`     | Whitelist servers registering on the server list, format `ip address 1[;ip address 2]` | (central server only) [See note](Choosing-a-Server-Type#3-central). |
```
Comment (gilgongo) [inline wiki/en/en-Command-Line-Options.md:34]: As per the brief discussion on this, we should change this to say "Allow only listed servers to register on the server list..." I think.

File: 1-index.html (hunk 1 above)
Comment (ann0see) [reply 1-index.html:19]: Probably that's something for the changes branch since it needs translation.

**Comment by jujudusud** [comment]
Updating general.yml and navigation.yml in PR (#434)

**Pull request merged**